site/src/docs/backup/backup/index.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Why wouldn't you put this in `site/src/docs/backup/index.md` file?

Reply:
Categories are not links and it's not possible to go to them from the interface as far as I can see.